src/Checker/AdminChecker.php

```diff
@@ -32,10 +32,32 @@ public function isMaintenance(MaintenanceConfiguration $configuration, Request $
         /** @var string $adminPrefix */
         $adminPrefix = $this->params->get('sylius_admin.path_name');
 
+        if (str_starts_with($getRequestUri, '/_profiler') || str_starts_with($getRequestUri, '/_wdt')) {
```

Comment:
Shouldn't we create a ProfilerChecker instead ?

Reply:
Good idea ;)